schema.graphql

```diff
@@ -438,7 +438,7 @@ type Mutation {
   cancelMembershipRequest(membershipRequestId: ID!): MembershipRequest!
   checkIn(data: CheckInInput!): CheckIn!
   createAdmin(data: UserAndOrganizationInput!): User!
-  createAdvertisement(endDate: Date!, link: String!, name: String!, orgId: ID!, startDate: Date!, type: String!): Advertisement!
+  createAdvertisement(endDate: Date!, mediaUrl: URL, name: String!, orgId: ID!, startDate: Date!, type: String!): Advertisement!
```

Comment:
Why is `mediaUrl` not required? Can an advertisement be made without it?

Also in mutations it's better to have them accept a single input object and define an input graphql type for it, here for example something like `input CreateAdvertisementInput`, having seperate fields littered around like this just makes the schema design inconsistent.